omnigibson/examples/object_states/slicing_demo.py

```diff
@@ -89,6 +89,7 @@ def main(random_selection=False, headless=False, short_exec=False):
     knife.set_position_orientation(
         position=apple.get_position() + np.array([-0.15, 0.0, 0.2]),
         orientation=T.euler2quat([-np.pi / 2, 0, 0]),
+        # orientation=T.euler2quat([np.pi / 2, 0, 0])   # The back of the knife cuts the apple as well
```

Comment:
In tests/test_transition_rules.py create two unit tests, one that has the object falling on the sharp side and cutting, one that has the object falling on the blunt side and not cutting. Essentially you need to create a function called `test_{test_name_here}` and run it using `pytest tests/test_transition_rules.py` and it should pass. Ask for in-person help from Eric if you can't figure out how to run pytest. Once the test is ready, remove this change.